tests/unit/data_formats/coco/test_roboflow.py

```diff
@@ -2,12 +2,15 @@
 #
 # SPDX-License-Identifier: MIT
 
+from typing import Any, Dict, Optional, Type
```

Comment:
Type is imported but not used. Doesn't the commit-hook apply to the test code?

Reply:
The "tests" directory is excluded during the `ruff` check.

Reply:
Anyway I'll revert it with fixing the errors on Windows